src/pattern-library/components/PlaygroundApp.js

```diff
@@ -2,11 +2,13 @@ import classnames from 'classnames';
 
 import { Link, SvgIcon } from '../../';
 
-import { getRoutes } from '../routes';
+import { componentGroups, getRoutes } from '../routes';
```

Comment:
Changes in this module support a third level of navigation (via `componentGroups`) and extract out some local navigation-related components.